homework.md

```diff
@@ -92,19 +92,19 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send your code to Github
```

Comment:
I think this means to save your changes to the local repository. Have look again.